contents/javascript/VueJs/Intro.md

```diff
@@ -0,0 +1,133 @@
+# VueJs
+
+Author: [Lu Lechuan](https://github.com/LuLechuan)
+
+Vue.js is an open-source progressive [JavaScript framework](https://en.wikipedia.org/wiki/JavaScript_framework) for building user interfaces. It is designed to improve code quality and maintainablity. Simply put, it is used as a tool to write JavaScript fast and easy.
```

Comment:
Since you used VueJs, let's stick with that instead of Vue.js
Or you can choose to use Vue.Js in the entire document